src/styles/main.scss

```
@@ -0,0 +1,50 @@
+@import "./_keyframes";
+@import "./_mixin";
+
+.container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+}
+
+.container_clock {
+  position: absolute;
+  transform: translate(-50%, -50%);
+  top: 50%;
+  left: 50%;
+}
+
+.container_circle {
+  height: 20px;
+  width: 20px;
+  background-color: #4b9aaa;
+  border-radius: 50%;
+  position: relative;
+  bottom: 163px;
+  right: 8px;
+}
+
+.container_string-first {
+  position: relative;
+  height: 100px;
+  @include index(5px);
+  background-color: #d15559;
+  transform-origin: bottom left;
+  transform: rotate(40deg);
+  animation-name: name;
+  animation-duration: 360s;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
```

Not an necessity, but generally using shorthand version of rule looks better
```suggestion
  animation: name 360s linear infinite;
```